### PR TITLE
Implements the deploy feature discussed in #2.

### PR DESCRIPTION
**Pull request by simonedavico**

### CODE REVIEW

File: application/src/main/java/cloud/benchflow/experimentsmanager/ExperimentsManagerApplication.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import org.glassfish.jersey.media.multipart.MultiPartFeature;;
+
+public class ExperimentsManagerApplication extends Application<ExperimentsManagerConfiguration> {
+    public static void main(String[] args) throws Exception {
+        new ExperimentsManagerApplication().run(args);
+    }
+
+    @Override
+    public String getName() {
+        return "hello-world";
```
Comment (VincenzoFerme) [inline application/src/main/java/cloud/benchflow/experimentsmanager/ExperimentsManagerApplication.java]: Change it to the actual name of the application.

Comment (simonedavico) [inline application/src/main/java/cloud/benchflow/experimentsmanager/ExperimentsManagerApplication.java]: What do you mean? Simply `ExperimentsManager`?

Comment (VincenzoFerme) [inline application/src/main/java/cloud/benchflow/experimentsmanager/ExperimentsManagerApplication.java]: experiments-manager

Comment (simonedavico) [inline application/src/main/java/cloud/benchflow/experimentsmanager/ExperimentsManagerApplication.java]: Oh, you are referring to the `getName` method. Got it.

**Pull request merged**